sklearn_numba_dpex/common/tests/test_kernels.py

```diff
@@ -32,17 +32,28 @@
     ],
```

Comment:
Let's parametrize the test by shape tuple only and do the `dpt.arange` call in the body of the test instead. The expected result can be computed by numpy instead of passing it as a test parameter and using the transposition trick.

This way it will be easier to run the test for more shapes, including more edge cases such as `(1, 1)`, `(1, 2)` and `(3, 1)` and maybe some larger shapes.

Reply:
Adding edge cases.

The transpose trick is nice because we only have to define the test cases for one axis and copy for the other axis no ?

Reply:
I find it not nice because it's harder to reason about what is the intention of the test and what is contiguous in memory.

Reply:
I rewrote the test as you said